target/linux/ramips/mt7621/base-files/lib/upgrade/linksys.sh

```diff
@@ -0,0 +1,93 @@
+#
+# Copyright (C) 2014 OpenWrt.org
```

Comment:
You are not openwrt. But please add an SPDX license identifier instead.

Reply:
Good point, done!